src/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -494,11 +496,11 @@
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
     <Reference Include="System.Runtime.Numerics" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netfx' or '$(TargetGroup)' == 'net47'">
```

Comment:
Yeah I hate these duplicated checks as well I was actually thinking about adding a property IsNetFx or something similar but your right for this project IsPartialFacade works as well but that isn't true in all libraries. I will change these.